test/inspector/inspector-helper.js

```diff
@@ -0,0 +1,429 @@
+'use strict';
+const assert = require('assert');
+const common = require('../common');
+const fs = require('fs');
+const http = require('http');
+const path = require('path');
+const spawn = require('child_process').spawn;
+
+const TIMEOUT = 15 * 1000;
+
+const mainScript = path.join(
+  common.fixturesDir,
+  'loop.js'
+);
+
+function send(socket, message, id, callback) {
+  const msg = JSON.parse(JSON.stringify(message)); // Clone!
+  msg['id'] = id;
+  const message_buf = Buffer.from(JSON.stringify(msg));
+
+  const ws_header_buf = Buffer.allocUnsafe(16);
+  ws_header_buf.writeUInt8(0x81, 0);
+  let byte2 = 0x80;
+  const body_len = message_buf.length;
+  let mask_offset = 2;
+  if (body_len < 126) {
+    byte2 = 0x80 + body_len;
+  } else if (body_len < 65536) {
+    byte2 = 0xFE;
+    ws_header_buf.writeUInt16BE(body_len, 2);
+    mask_offset = 4;
+  } else {
+    byte2 = 0xFF;
+    ws_header_buf.writeUInt32BE(body_len, 2);
+    ws_header_buf.writeUInt32BE(0, 6);
+    mask_offset = 10;
+  }
+  ws_header_buf.writeUInt8(byte2, 1);
+  ws_header_buf.writeUInt32BE(0x01020408, mask_offset);
+
+  for (let i = 0; i < message_buf.length; i++) {
+    message_buf[i] = message_buf[i] ^ (1 << (i % 4));
+  }
```

Comment:
Not too sure if it's enforced in all the codebase but I think one-line for's and if's don't use brackets.